pypeman/channels.py

```diff
@@ -66,6 +67,10 @@ def __init__(self, name=None, parent_channel=None, loop=None, message_store_fact
         self._status = BaseChannel.STOPPED
         self.processed_msgs = 0
         self.interruptable_sleeper = Sleeper(loop)  # for interruptable sleeps
+        self.done_callback = None
```

Comment:
please explain exactly when a callback will be called and whether they are mutually exclusive or whether it is possible that two or more callbacks can be called for the same node under certain circumstances.

Reply:
also document call order if multiple callbacks can be called